data/hai/hai reveal 5 philosophers.txt

```diff
@@ -588,9 +595,8 @@ mission "Hai Reveal [B07-long] Learn of Keystones"
 	clearance
 	to offer
 		has "Hai Reveal [B06-long] Tarazed Analysis: active"
-	source
-		government "Smuggler (Hai Trafficker)"
 	on offer
+		fail "Hai Reveal [B06-long] Tarazed Analysis"
```

Comment:
Fix already PR'd #7454

Reply:
You fixed this after I tested it, so I'll retest this part of the campaign with your fix instead.